src/week2_CountInversions/PrintArray.java

```diff
@@ -0,0 +1,10 @@
+package week2_CountInversions;
+
+public class PrintArray {
```

Comment:
Special class not required for printing array. Even if one required, name needs to be meaningful. You can create a utility class and a method inside it that does the printing